static/style.css

```diff
@@ -246,3 +246,97 @@ p.text-black.text-base.company-blurb {
 a.underline, span.underline {
     text-underline-offset: 3px;
 }
+
+#people-page-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+    grid-auto-rows: 135px;
+    grid-auto-flow: dense;
+    gap: 10px;
+    margin-top: 30px;
+    margin-bottom: 30px;
+}
+
+#people-page-grid > * {
+    grid-area: span 1 / span 1;
+}
+
+#people-page-grid .tile-2x2 {
+    grid-area: span 2 / span 2;
+}
+
+@media (max-width: 470px) {
+    #people-page-grid .tile-2x2 {
+        grid-area: span 2 / span 1;
+    }
+}
+
+#people-page-grid .tile-1x2 {
+    grid-area: span 2 / span 1;
+}
+
+.person-tile {
+    justify-self: stretch;
+    background-color: #f3f3f3;
+    max-height: 135px;
+    min-width: 240px;
+    overflow: auto;
+    transition: transform 0.3s;
+}
+
+.person-tile .person-name {
+    margin-top: 10px;
+    font-size: 0.7em;
+}
+
+.person-tile .shortbio {
+    margin-top: 10px;
+    align-items: baseline;
+    font-size: 0.7em;
```

Comment:
let's use tailwind standard classes for specifying font sizes

Reply:
Not going to do this in this PR, since I realised we don't have the tailwind library at all, and I need that custom size.

Please feel free to open this in a separate issue, I think the tailwind part needs a separate rework of sorts anyway.

Reply:
Cool cool, we dont have to do this via tailwind. Can put in the relative sizes but this is not a blocker and can be taken up later if needed.